src/main/java/com/Novi/TechItEasy/controllers/TelevisionController.java

```diff
@@ -0,0 +1,56 @@
+package com.Novi.TechItEasy.controllers;
+
+import com.Novi.TechItEasy.exceptions.NameTooLongException;
+import com.Novi.TechItEasy.exceptions.RecordNotFoundException;
+import jakarta.websocket.server.PathParam;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+@RestController
+public class TelevisionController {
+
+    String[] televisionBrands = {"Sony", "Phillips", "LG"};
+    List<String> televisionDatabase = new ArrayList<>(List.of(televisionBrands));
+
+
+
+    @GetMapping(value ="/televisions")
+    public ResponseEntity<List<String>> getTelevisions() {
+        return ResponseEntity.ok(televisionDatabase);
+    }
+
+    @GetMapping(value ="/television/{id}")
+    public ResponseEntity<String> getTelevision(@PathVariable int id) {
+        if (id >= televisionDatabase.size()) {
+            throw new IndexOutOfBoundsException();
```

Comment:
Deze check is onnodig, omdat je deze exception al afvangt in je exceptioncontroller. Wanneer je dus een tv uit de lijst wilt halen met index 1000, gooit java zo'n IndexOutOfBoundsException op en vang jij die af in je exceptionhandler.